tests/test_socket_server.py

```diff
@@ -28,6 +28,8 @@
     format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.DEBUG
 )
 
+TIMEOUT = 5
```

Comment:
Good addition. Please also add a brief code comment that includes units.

Reply:
Done.